internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -727,3 +727,24 @@ func (r *Reconciler) reconcileReferencedTemplate(ctx context.Context, in reconci
 
 	return nil
 }
+
+// CreateError removes details about the specific name of the object from the Error message. The full names include a hash
+// Cluster conditions.
+func createError(err error, obj client.Object) error {
```

Comment:
q: should this func goes in the log package

what about
```suggestion
func apiErrorWithoutObjectName(err error, obj client.Object) error {
```

Reply:
I don't think it should be in the log package as it's not directly related to logs. The error is only logged at some point by controller runtime.

Reply:
I think it should live in this package unless we've got some other place it could be used. Right now it's just a single purpose function so I don't see a reason to move it.